### PR TITLE
Linode Entity Detail Adjustments

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
- Update the Linode Entity Detail used in the Linode Detail and Linode Landing grid view pages
- Clean up the component and remove unnecessary styles
- Also added a fix to prevent the status on Domain Landing from wrapping 

<details><summary>Screenshots</summary>

### **Before:**
Linode Detail View
Screen Shot 2020-11-25 at 2 50 51 PM

Linode Landing Grid View
Screen Shot 2020-11-25 at 2 50 40 PM

### **After:**
Linode Detail View
Screen Shot 2020-11-25 at 2 48 29 PM

Linode Landing Grid View
Screen Shot 2020-11-25 at 2 48 44 PM
</details>

## Note to Reviewers
~There's definitely some scope creeping happening here so will wait for [https://github.com/linode/manager/pull/7145](https://github.com/linode/manager/pull/7145) to be merged first so I can fix the conflicts~

### CODE REVIEW

**Review by WilkinsKa1 — COMMENTED** [review]
I know this has the wait label, but initial review on this looks great- this was a huge undertaking, really fantastic work on this. 

Only thing I've noticed so far is on linode landing details variant, the label is a bit wonky:
Screen Shot 2020-11-24 at 4 38 55 PM

I think we can potentially remove the entity icon to regain some space and will probably need some additional flex adjustments to play well with the status.

**Review by johnwcallahan — COMMENTED** [review]
Do we have a design for tablet viewport sizes? There's a lot of whitespace here:

Screen Shot 2020-11-30 at 12 46 18 PM

There's also a regression in VLANS: 
Screen Shot 2020-11-30 at 12 53 11 PM

And (less importantly) DBaaS:
Screen Shot 2020-11-30 at 12 57 27 PM

**Comment by tiffwong** [comment]
> Do we have a design for tablet viewport sizes? There's a lot of whitespace here:
> 
> Screen Shot 2020-11-30 at 12 46 18 PM
> 
> There's also a regression in VLANS:
> Screen Shot 2020-11-30 at 12 53 11 PM
> 
> And (less importantly) DBaaS:
> Screen Shot 2020-11-30 at 12 57 27 PM

There are currently no tablet designs but I'll inquire about the status of those and I'll create a new ticket to fix the regressions

**Review by Jskobos — CHANGES_REQUESTED** [review]
This looks great, _very_ different from the CMR of yore. Everything seems to be working and I'd suggest getting this in to develop asap and iterating. That said, one blocking change (Dark theme needs some tweaking):
Screen Shot 2020-12-01 at 9 09 13 AM

Optionally, a fixed height for the footer would be nice. There's a bit of jump when opening the tag input.

And I noticed that the node pools table for LKE still has the old row click. There should be a separate task to make the label a link and remove the row-wide click handler.

**Review by DevDW — COMMENTED** [review]
Nice job with this; I noticed a couple of things in addition to what others have observed:

- On the Managed page, should the "Open Support Ticket" and "Docs" button be different colors?

Screen Shot 2020-12-01 at 10 51 35 AM

- I think we should change "Vlans" to "VLANs". This will make it consistent with how we're capitalizing it for the Create button here and elsewhere in the app.

Screen Shot 2020-12-01 at 10 52 28 AM

File: packages/manager/src/features/Domains/DomainDetail/DomainDetail.tsx [hunk 1]
```diff
@@ -115,22 +115,30 @@ const DomainDetail: React.FC<CombinedProps> = props => {
 
   return (
     <React.Fragment>
-      <Grid container justify="space-between" className="m0">
+      <Grid
```
[the remark this answers is not in the document]
Comment (tiffwong) [reply packages/manager/src/features/Domains/DomainDetail/DomainDetail.tsx]: Not sure what happened but I replaced it with the `TagsPanelRedesigned` and that seems to fix the issue

**Comment by tiffwong** [comment]
@DevDW good catch! There seem to be some inconsistencies throughout the app when it comes to the link colors in the dark theme so I'll create a separate ticket to handle any others but I did fix the managed one

You make a good point with the VLAN naming convention so I'll include that in another PR that I'll have up soon

**Review by Jskobos — APPROVED** [review]
Thanks for the quick turnaround. I found one other small thing, but that can go in a follow-on:

Screen Shot 2020-12-01 at 1 29 32 PM

(the header bulges out a pixel or two, happens on both themes but easier to see on dark)

**Review by johnwcallahan — APPROVED** [review]

**Review by DevDW — APPROVED** [review]

**Pull request merged**